config/clock.rb

```diff
@@ -24,6 +22,11 @@ class Clock
   every(1.hour, 'SendChaseEmailToProviders', at: '**:35') { SendChaseEmailToProvidersWorker.perform_async }
   every(1.hour, 'SendChaseEmailToCandidates', at: '**:40') { SendChaseEmailToCandidatesWorker.perform_async }
   every(1.hour, 'UpdateFeatureMetricsDashboard', at: '**:45') { UpdateFeatureMetricsDashboard.perform_async }
+  every(1.hour, 'SyncNewCycleIfRequired', at: '**:05') do
```

Comment:
can we tie this to exactly 5 minutes after midnight `00:05` rather than `**:05`?

Reply:
So for now we can just run this daily and check for 
`Time.zone.now.between?(CycleTimetable.find_closes, CycleTimetable.find_closes + 1.day)`